docs/components/rule.md

```diff
@@ -81,7 +81,9 @@ The most important metrics to alert on are:
 indicate connection, incompatibility or misconfiguration problems.
 
 * `prometheus_rule_evaluation_failures_total`. If greater than 0, it means that that rule failed to be evaluated, which results in
-either gap in rule or potentially ignored alert. Alert heavily on this if this happens for longer than your alert thresholds.
+either gap in rule or potentially ignored alert. This metric might indicate problems on the queryAPI endpoint you use.
+
```

Comment:
This new line here breaks the formatting of the list. This makes the line “alert heavily...” appear out of the list, and then the list continues.